<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(117, 117, 117);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#predictButton{	
	
background-color: qlineargradient(spread:pad, x1:0.489263, y1:1, x2:0.494, y2:0.00590909, stop:0 rgba(93, 173, 1, 255), stop:0.489474 rgba(96, 152, 0, 255), stop:1 rgba(100, 185, 1, 255));

	
	color: rgb(203, 255, 204);
    border-radius:8px;
border:0.5px solid rgba(106, 167, 0, 255);
font: 75 15pt &quot;Segoe UI&quot;;

}

#predictButton:hover{	

	background-color: qlineargradient(spread:pad, x1:0.489263, y1:1, x2:0.494, y2:0.00590909, stop:0 rgba(101, 188, 1, 255), stop:0.489474 rgba(106, 167, 0, 255), stop:1 rgba(107, 198, 1, 255));
	
border:0.5px solid  rgba(93, 173, 1, 255);

}

#predictButton:pressed{	
	background-color: qlineargradient(spread:pad, x1:0.489263, y1:1, x2:0.494, y2:0.00590909, stop:0 rgba(93, 172, 1, 255), stop:0.489474 rgba(93, 147, 0, 255), stop:1 rgba(95, 177, 1, 255));
}

QListWidget{
	background-color: qlineargradient(spread:pad, x1:0.542, y1:1, x2:0.542, y2:0.722, stop:0 rgba(213, 213, 213, 255), stop:0.273684 rgba(226, 226, 226, 255), stop:0.563158 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;
border-bottom-left-radius:8px;
border-bottom-right-radius:8px;
}

#label_3,#label_4{
border-top-left-radius:8px;
border-top-right-radius:8px;
font: 10pt &quot;Segoe UI&quot;;	
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.526, y1:0.267, x2:0.526053, y2:1, stop:0 rgba(77, 77, 77, 255), stop:0.536842 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
}

QSrollBar:horizontal{
border: 1px solid #999999;
width:8px;
background:white;
margin: 0px 0px 0px 0px;
}

QSrollBar:vertical{
border: 1px solid #999999;
width:8px;
background:white;
margin: 0px 0px 0px 0px;
}

 QScrollBar::handle:horizontal{
width:8px;
background:black;
border-radius:4px;
min-height:0px;

}

 QScrollBar::handle:vertical{
width:8px;
background:black;
border-radius:4px;
min-height:0px;

}

#deselct_btn , #selct_btn{
 border-radius:5px;
border:0.6px solid black;
background-color: rgb(0, 0, 235);
color: rgb(255, 255, 255);
} 

#deselct_btn:hover,#selct_btn:hover{
 border-radius:5px;
border:0.6px solid black;
background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
} 

#deselct_btn:pressed,#selct_btn:pressed{
 border-radius:5px;
border:0.6px solid black;
background-color: rgb(0, 0, 205);
color: rgb(255, 255, 255);
} 

#kg{
border-radius:5px;
border:0.5px solid gray;
}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Segoe UI&quot;;
background-color: qradialgradient(spread:pad, cx:0.542, cy:0, radius:0.729, fx:0.568, fy:0, stop:0 rgba(172, 172, 172, 255), stop:1 rgba(117, 117, 117, 255));</string>
    </property>
    <property name="text">
     <string>Disease Prediction System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="symp_List">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="sel_Symp_List">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>90</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QToolButton" name="deselct_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="selct_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kg">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>293</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>293</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Your Weight(in Kgs)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predictButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>69</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Symptoms</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>70</y>
      <width>256</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Selected Symptoms</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>722</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.522, y1:0, x2:0.528, y2:1, stop:1 rgba(175, 175, 175, 255), stop:0.5 rgba(117, 117, 117, 255));

selection-background-color:qlineargradient(spread:pad, x1:0.526, y1:0.6875, x2:0.536579, y2:1, stop:0 rgba(117, 117, 117, 255), stop:0.989474 rgba(160, 160, 160, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>720</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>5</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(117, 117, 117, 255), stop:0.485149 rgba(255, 255, 255, 255), stop:1 rgba(117, 117, 117, 255));</string>
    </property>
    <property name="lineWidth">
     <number>-1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>4</y>
      <width>16</width>
      <height>493</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Segoe UI&quot;;
selection-background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
background-color: rgb(227, 227, 227);</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>5</y>
      <width>557</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QFrame" name="image">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>365</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
